rustls-libssl/tests/config.c

```diff
@@ -14,38 +14,46 @@ static const int conf_flags[] = {SSL_CONF_FLAG_SERVER, SSL_CONF_FLAG_CLIENT,
 
 #define NUM_FLAGS (sizeof(conf_flags) / sizeof(conf_flags[0]))
 
-static const char *supported_cmds[] = {"-min_protocol",
-                                       CUSTOM_PREFIX "min_protocol",
-                                       "MinProtocol",
-                                       CUSTOM_PREFIX "MinProtocol",
-
-                                       "-max_protocol",
-                                       CUSTOM_PREFIX "max_protocol",
-                                       "MaxProtocol",
-                                       CUSTOM_PREFIX "MaxProtocol",
-
-                                       "VerifyMode",
-                                       CUSTOM_PREFIX "VerifyMode",
-
-                                       "-cert",
-                                       CUSTOM_PREFIX "cert",
-                                       "Certificate",
-                                       CUSTOM_PREFIX "Certificate",
-
-                                       "-key",
-                                       CUSTOM_PREFIX "key",
-                                       "PrivateKey",
-                                       CUSTOM_PREFIX "PrivateKey"
-
-                                                     "-verifyCApath",
-                                       CUSTOM_PREFIX "verifyCApath",
-                                       "VerifyCAPath",
-                                       CUSTOM_PREFIX "VerifyCAPath",
-
-                                       "-verifyCAfile",
-                                       CUSTOM_PREFIX "verifyCAfile",
-                                       "VerifyCAFile",
-                                       CUSTOM_PREFIX "VerifyCAFile"};
+static const char *supported_cmds[] = {
```

Comment:
ahah, it's back being sensible. is it the length that controls this? or the trailing comma?

Reply:
> is it the length that controls this? or the trailing comma?

I don't think it's the trailing comma. I've been unable to determine the formatting logic (or how to influence it). The results are UGLY in all instances :sob: